tests/algorithm/scripts/calculate_metrics.py

```diff
@@ -0,0 +1,90 @@
+RESULTS_FILE = "output.csv"
```

Comment:
I think this variable can be deleted now that its a function param

Reply:
Oh yep, you're totally right!